composer.lock

```diff
@@ -4020,5 +3982,5 @@
     "platform-overrides": {
         "php": "7.4"
     },
-    "plugin-api-version": "2.2.0"
+    "plugin-api-version": "2.1.0"
```

Comment:
Going backwards?

Reply:
I'm not sure how this happened. I ran `composer require --dev -W nextcloud/coding-standard` to update cs-fixer an make the check actually work.

Reply:
Done. I reverted it to 2.2.0.